src/routes/jira/events/jira-select-version-router.ts

```diff
@@ -1,6 +1,6 @@
 import { Router } from "express";
 import { JiraJwtTokenMiddleware } from "middleware/jira-jwt-middleware";
-import { JiraSelectVersionGet } from "../select/jira-select-version-get";
+import { JiraSelectVersionGet } from "../server/jira-select-version-get";
```

Comment:
Renamed this to server as it makes more sense to keep all the GHE server routes in the one dir.